web/src/lib/components/faces-page/people-card.svelte

```diff
@@ -73,22 +66,16 @@
     {/if}
   </a>
 
-  <div class="absolute right-2 top-2" class:hidden={!showVerticalDots}>
-    <CircleIconButton
-      color="opaque"
+  <div class="absolute top-2 right-2">
+    <ButtonContextMenu
+      buttonClass="icon-white-drop-shadow focus:opacity-100 {showVerticalDots ? 'opacity-100' : 'opacity-0'}"
+      buttonColor="opaque"
+      buttonPadding="2"
+      buttonSize="20"
       icon={mdiDotsVertical}
       title={$t('show_person_options')}
-      size="20"
-      padding="2"
-      class="icon-white-drop-shadow"
-      on:click={showMenu}
-    />
-  </div>
-</div>
-
-{#if showContextMenu}
-  <Portal target="body">
-    <ContextMenu {...contextMenuPosition} onClose={() => onMenuExit()}>
+      usePortal
```

Comment:
Is this actually ever required? It seems like the context menu should always show above/on top of the clicked button.

Reply:
Yes - I added the `Portal` to get the People card and External Library settings dropdown working as expected. They used a portal before this change and the dropdown menu does not render properly without it in both of these scenarios.

Reply:
I found a way to remove the need for a portal - set the positioning of the `<ContextMenu>` as `fixed` rather than `absolute`.